Packs/Netskope/Integrations/NetskopeEventCollector/NetskopeEventCollector.py

```diff
@@ -91,126 +72,75 @@ def v1_get_alerts_request(self, last_run: dict, limit: Optional[int] = None) ->
-
-
-def date_to_seconds_timestamp(date_str_or_dt: Union[str, datetime]) -> int:  # pragma: no cover
-    """
-    Converts date string or datetime object to a timestamp in seconds.
-
-    Args:
-        date_str_or_dt (Union[str, datetime]): The datestring or datetime.
-
-    Returns:
-        int: The timestamp in seconds.
-    """
-
-    return date_to_timestamp(date_str_or_dt) // 1000
-
-
 def get_sorted_events_by_type(events: list, event_type: str = '') -> list:
     filtered_events = [event for event in events if event.get('event_type') == event_type]
-    filtered_events.sort(key=lambda k: k.get('_id'))
+    filtered_events.sort(key=lambda k: k.get('timestamp'))
     return filtered_events
 
 
-def get_last_run(events: list, last_run: dict) -> dict:  # type: ignore
+def create_last_run(events: list, last_run: dict) -> dict:  # type: ignore
     """
     Args:
     events (list): list of the event from the api
     last_run (dict): the dictionary containing the last run times for the event types
     Returns:
     A dictionary with the times for the next run
     """
-    alerts = get_sorted_events_by_type(events, event_type='alert')
-    audit_events = get_sorted_events_by_type(events, event_type='audit')
-    applications_events = get_sorted_events_by_type(events, event_type='application')
-    network_events = get_sorted_events_by_type(events, event_type='network')
-    if not alerts:
-        alerts_time = last_run['alert']
-    else:
-        alerts_time = alerts[-1]['timestamp']
-    if not applications_events:
-        applications_time = last_run['application']
-    else:
-        applications_time = applications_events[-1]['timestamp']
-    if not audit_events:
-        audit_time = last_run['audit']
-    else:
-        audit_time = audit_events[-1]['timestamp']
-    if not network_events:
-        network_time = last_run['network']
-    else:
-        network_time = network_events[-1]['timestamp']
-    return {'alert': alerts_time, 'application': applications_time, 'audit': audit_time,
-            'network': network_time}
+    for event_type in ['alert', 'audit', 'application', 'network']:
+        ordered_events_by_type = get_sorted_events_by_type(events, event_type)
+        events_time = ordered_events_by_type[-1]['timestamp'] if ordered_events_by_type else last_run[event_type]
+        last_run[event_type] = events_time
+    return last_run
 
 
 ''' COMMAND FUNCTIONS '''
 
 
 def test_module(client: Client, api_version: str, last_run: dict) -> str:
-    if api_version == 'v1':
-        last_run = {'alert': 604800}  # For v1 it supports only specific timestamps - - Last 30 days
-        response = client.v1_get_alerts_request(last_run, limit=1)
-    else:
-        response = client.get_events_request_v2(last_run, limit=1, is_test=True)
 
-    if response:
+    events = fetch_events_command(client, api_version, last_run, max_fetch=1)
+    if events:
```

Comment:
If the system doesn't have any events, the list will be empty, and we will return an error. I don't think we need to check the return value of the fetch since if the `fetch_events_command` will fail you will get an exception right?